src/main/java/lotto/domain/LottoRankingSystem.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum LottoRankingSystem {
```

Comment:
```suggestion
public enum LottoRank {
```
`RankingSystem`까지 가기에는 시스템을 뜻하는 객체는 아닌 것 같고 간단하게 `LottoRank`로도 충분해보여요 :)